chromadb/test/api/test_collection.py

```diff
@@ -0,0 +1,59 @@
+from chromadb.api import ClientAPI
+from chromadb.test.conftest import NOT_CLUSTER_ONLY
+import pytest
+
+
+def duplicate_collection_create(
+    client: ClientAPI,
+) -> None:
+    collection = client.create_collection(
+        name="test",
+        metadata={"hnsw:construction_ef": 128, "hnsw:search_ef": 128, "hnsw:M": 128},
+    )
+
+    try:
+        client.create_collection(
+            name="test",
+            metadata={
+                "hnsw:construction_ef": 128,
+                "hnsw:search_ef": 128,
+                "hnsw:M": 128,
+            },
+        )
+        assert False, "Expected exception"
+    except Exception as e:
+        print("Collection creation failed as expected with error ", e)
+        assert "already exists" in e.args[0]
+
+
+def not_existing_collection_delete(
+    client: ClientAPI,
+) -> None:
+    try:
+        collection = client.delete_collection(
+            name="test101",
+        )
+        assert False, "Expected exception"
+    except Exception as e:
+        print("Collection deletion failed as expected with error ", e)
+        assert "does not exist" in e.args[0]
+
+
+def test_duplicate_collection_create_local(client: ClientAPI) -> None:
```

Comment:
I don't think you need this function. You can just have `duplicate_collection_create` and `not_existing_collection_delete`. The [client fixture](https://github.com/chroma-core/chroma/blob/00992b2d67c08be4dc99bad6d955f7f7ca715750/chromadb/test/conftest.py#L780) includes a[ basic http client fixture  ](https://github.com/chroma-core/chroma/blob/00992b2d67c08be4dc99bad6d955f7f7ca715750/chromadb/test/conftest.py#L603) if `CHROMA_CLUSTER_TEST_ONLY` is set. 

This github action https://github.com/chroma-core/chroma/blob/00992b2d67c08be4dc99bad6d955f7f7ca715750/.github/workflows/_python-tests.yml#L87 sets the var - so we could add test/api to that list

Reply:
ty kindly! This is much cleaner!